lib/ssh.js

```diff
@@ -262,6 +262,12 @@ function _buildCommand (args) {
   return cmd.trim()
 }
 
+function _connectionDefaults() {
+  return {
+    port: 80
+  }
+}
```

Comment:
I haven't done enough javascript to know if this is the best way to do this (the whole thing - the extraction into a function and then use of `...` operator to merge them in).